ci-operator/config/kiegroup/kie-cloud-tests-container/kiegroup-kie-cloud-tests-container-main__rhba-ocp4.14-lp-interop.yaml

```
@@ -0,0 +1,51 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: rhba-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: "1"
+      memory: 2Gi
+tests:
+- as: rhba-interop-aws
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "rhba-interop-tests", "failure_type": "pod_failure", "classification": "Test Execution", "jira_project": "LPTOCPCI"},
+          {"step": "rhba-interop-tests", "failure_type": "test_failure", "classification": "OCP Interop Test Failure", "jira_project": "RHPAM", "jira_component": "Cloud", "jira_additional_labels": "ocp-interop"}
```

```suggestion
          {"step": "rhba-interop-tests", "failure_type": "test_failure", "classification": "OCP Interop Test Failure", "jira_project": "RHPAM", "jira_component": "Cloud", "jira_additional_labels": ["ocp-interop"]}
```

You'll need to format the additional labels like this, as it is a list of values. Here is the documentation for that value: https://github.com/CSPI-QE/firewatch/blob/main/docs/configuration_guide.md#jira_additional_labels